src/utils/computedReport/getComputedReportItems.ts

```diff
@@ -96,7 +97,10 @@ export function getUnsortedComputedReportItems<
             : '';
 
         // org_unit_id workaround for storage and instance-type APIs
-        const id = idKey === 'org_entities' ? value.id || value.org_unit_id : value[idKey];
+        const id =
+          idKey === 'org_entities'
+            ? value.id || value.org_unit_id
+            : value[idKey];
```

Comment:
I have no idea. I have a feeling it's related to `prettier` and its configuration file. For some reason I do not see that configuration and I assumed it was removed for obvious reasons.

Now I noticed that `prettier` is not installed (it's not in `package.json`) - do you have an idea why since you did the last change there.